src/pages/FeedbackPage.tsx

```diff
@@ -124,7 +124,7 @@ async function getUsersByComments(comments: TComment[]) {
   return groupBy(user, 'id') as Record<string, User[]>
 }
 
-async function fetchDataByFeedbackId(id?: string) {
+export async function fetchDataByFeedbackId(id?: string) {
```

Comment:
Maybe let's move it to services/feedback or kind of that as it's being used in multiple places?